### PR TITLE
docs: add Google Analytics tracking

### PR DESCRIPTION
**Pull request by darryltec**
## Motivations

Because we want to know how much traffic we getting and where

### Added

- installed `react-ga4` package instead of @storybook/google-analytics since it doesn't support GA4.
- Tracked page visits through manager js
- Also included errors since that's what the @storybook/google-analytics does 

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

- Install this [chrome extension](https://chrome.google.com/webstore/detail/google-analytics-debugger/jnkmfdileelhofjcijamephohjechhna?hl=en)
- Ensure your browser extensions that blocks trackers are disabled
- Once done, click the google analytics extension to turn it on. It should have an ON badge
- It should start console logging the infos when viewing pages

Or just trust that it works

image
image

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: .storybook/manager.js [hunk 1]
```diff
@@ -39,3 +41,27 @@ addons.register("code/tab", () => {
     render: ({ key, active }) => <>{active && <Playground key={key} />}</>,
   });
 });
+
+addons.register('google-analytics', (api) => {
+  ReactGA.initialize("G-V1N3TQVQB5");
```
Comment (rodrigoeidelvein) [inline .storybook/manager.js:46]: Should we put it in a environment variable?

**Review by rodrigoeidelvein — COMMENTED** [review]
Maybe we need to do a environment check and only send the tracking events in the production environment

File: .storybook/manager.js (hunk 1 above)
Comment (darryltec) [reply .storybook/manager.js:46]: it's smart not to. GA tracking would only really track when it matches the URL of atlantis.getjobber.com

Comment (rodrigoeidelvein) [reply .storybook/manager.js:46]: Nice! 

And about this GA code, should we move it to a environment variable or is it okay to keep it public?

Comment (darryltec) [reply .storybook/manager.js:46]: okay to be public. it's pretty much gonna be public no matter what since it'll show up in the DOM

**Review by rodrigoeidelvein — APPROVED** [review]
Worked well using the extension, the events are showing up on my console!

LGTM!

**Pull request merged**